src/main/java/org/spongepowered/api/item/inventory/InventoryProperties.java

```diff
@@ -53,20 +57,23 @@
 
     /**
      * Represents the capacity of a inventory.
+     * TODO not present in impl | do we need this for query? {@link Inventory#capacity()} exists
```

Comment:
No, you have the Inventory object you can figure this out straight away with getting a property.

Reply:
* without

Reply:
Keep in mind that properties also serve a purpose for queries, so even if this may be duplicate for getting the capacity, it's still useful to use for querying for inventories with a specific capacity.

The same applies to the other properties.